Python/libraries/recognizers-date-time/recognizers_date_time/date_time/french/dateperiod_extractor_config.py

```diff
@@ -209,6 +209,14 @@ def week_day_of_month_regex(self) -> Pattern:
     def all_half_year_regex(self) -> Pattern:
         return self._all_half_year_regex
 
+    @property
+    def region_title_regex(self) -> Pattern:
```

Comment:
This should not happen. We should not add such new attributes to all languages, unless it applies in most.
Can it be changed to affect only CJK?